linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -29,8 +29,149 @@ use tonic_health::pb::{
 };
 use tracing::{info, warn};
 
-/// The endpoint used for all the tests
-const END_TO_END_STORAGE_SERVICE_ENDPOINT: &str = "127.0.0.1:8742";
+use async_lock::RwLock;
```

Comment:
See remark above.
Otherwise, yes, it would be nice to separate the testing code from the non-testing code. But the code is enmeshed right now.
I have several issues to unwind:
* The acceleration of the CI.
* The additional parallelization of the CI.
* Separation of test/non test code.

That cannot be done all at once.